rel/ci-sys.config

```diff
@@ -9,14 +9,10 @@
                                            ,{formatter, lager_default_formatter}
                                            ,{formatter_config, [time," [",severity,"] ", "|", {callid, <<"0000000000">>}, "|", module, ":", line, " (",pid, ") ", message, "\n"]}
                                            ]}
-                     ,{lager_file_backend, [{file, "log/console.log"}, {level, info}, {size, 10485760}, {date, "$D0"}, {count, 5}
+                     ,{lager_file_backend, [{file, "log/debug.log"}, {level, debug}, {size, 10485760}, {date, "$D0"}, {count, 5}
```

Comment:
@jamesaimonetti not sure how its done but we should add this file (log/debug.log) to ci configuration to have it available as part of the build outputs.